lib/Semantics/mod-file.h

```diff
@@ -8,8 +8,8 @@
 
 #ifndef FORTRAN_SEMANTICS_MOD_FILE_H_
 #define FORTRAN_SEMANTICS_MOD_FILE_H_
-
 #include "flang/Semantics/attr.h"
+#include "llvm/Support/raw_ostream.h"
 #include <sstream>
```

Comment:
Can we remove this header?

Reply:
This one is ok, as well as #include <sstream>.